bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,340 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "apm.nxiq.com/v1alpha1",
```

Comment:
strange package name. Maybe not urgent to fix this now, but at some point.

Reply:
it's consistent with the nxrm package names, and i believe it follows the initial patterns that yanai set. I'm not sure the overall significance of this package name yet, but I'd be willing to experiment with changing it at some point in the future.

Reply:
we use `nexusiq` elsewhere which feels like it's closer to our intended branding

Reply:
While I'm waiting for some feedback from Red Hat, I'll try renaming this.  It opens up larger questions like:
* should this be unique across all packages, or is it only meant to be unique combined with the kind?
* should it be like java packages and "sonatype" should be in there?

Reply:
looking at lots of other live examples out in the marketplace, it looks like people are using plain domain names, and it's then unique (* mostly) by the kind combined with the group. I'll give that a try. Thanks for the push.

Reply:
I think I'll do it in a new branch, though, since it's further shuffling of all the filenames, and i'll want to replicate it on the other operator.